holoviews/ipython/__init__.py

```diff
@@ -33,7 +33,9 @@ def __getattr__(attr):
         return IPTestCase
     raise AttributeError(f"module {__name__!r} has no attribute {attr!r}")
 
-
+HOLOVIEWS_B64_LOGO = (
```

Comment:
Can we keep this in the HTML, and have an option to overwrite it?

Reply:
Done in https://github.com/holoviz/holoviews/pull/6586/commits/b6693008d9e3d1f70ba54a4f1e4e207f9ca805f7